package.json

```diff
@@ -13,125 +13,80 @@
     "url": "https://github.com/digipolisantwerp/acpaas-ui_angular"
   },
   "scripts": {
-    "bootstrap": "node scripts/bootstrap.js",
-    "package": "node scripts/generate/package.js",
-    "styleguide": "ng serve styleguide",
-    "start:docker": "ng serve styleguide --host=0.0.0.0",
-    "prestart": "npm run bootstrap && npm run build && npm run examples",
-    "start": "npm run styleguide",
     "ng": "ng",
-    "lint": "ng lint",
+    "start": "ng serve --aot",
+    "build": "ng build",
+    "build:all": "node ./scripts/build-all.js",
+    "build:prod": "ng build --prod --base-href \"https://digipolisantwerp.github.io/acpaas-ui_angular/\"",
     "test": "ng test",
-    "e2e": "ng e2e",
-    "build": "node scripts/build.js",
-    "build:package": "ng build $example && npm run examples",
-    "postbuild": "node scripts/prepublish.js",
-    "examples": "node scripts/examples.js",
-    "example": "node scripts/generate/example.js",
-    "bump": "node scripts/version.js",
-    "deploy": "npm run deploy:patch",
```

Comment:
Er zullen wat hulpmiddelen verdwijnen optrent publishing, hebben jullie hier een alternatief voor of wordt alles goed gedocumenteerd zodat het publicatie process eenvoudig blijft?